src/components/AssetMeta/components/DocumentTable.tsx

```diff
@@ -65,19 +67,22 @@ export class DocumentTable extends React.PureComponent<
       docTypes,
       noDocumentsSign,
       collapseProps,
+      customCategorySort,
     } = this.props;
     const documentsByCategory = getDocumentsByCategory(
       docs || [],
       unknownCategoryName,
       docTypes,
       documentTypeField
     );
-    const categoryByPriority = getCategoryByPriority(
-      documentsByCategory,
-      categoryPriorityList
-    );
+    const {
+      categories,
+      prioritizedCount,
+    }: { categories: string[]; prioritizedCount: number } = customCategorySort
+      ? getCategoryByCustomSort(documentsByCategory, customCategorySort)
```

Comment:
Could we have prioritized array always in a game? Like:
1. Apply prioritized rules from `categoryPriorityList` (default or provided)
2. Then sort others by default/custom sort function

Reply:
done